src/main/java/com/terraformersmc/terrestria/init/TerrestriaGeneration.java

```diff
@@ -62,6 +34,7 @@ public static void init(BiomeConfig config, Set<String> enabledBiomes) {
 		addContinentalBiome(SAKURA_FOREST, OverworldClimate.TEMPERATE, 1.0, "sakura_forest", config, enabledBiomes);
 		addContinentalBiome(SNOWY_HEMLOCK_FOREST, OverworldClimate.SNOWY, 2.0, "snowy_hemlock_forest", config, enabledBiomes);
 		addBiomeVariant(Biomes.DEEP_OCEAN, VOLCANIC_ISLAND_SHORE, 0.10, "volcanic_island", config, enabledBiomes);
+		addBiomeVariant(Biomes.DESERT, DUNES, 0.2, "dunes", config, enabledBiomes);
```

Comment:
We should use addContinentalBiome instead, addBiomeVariant should only really be used for special cases in generation (such as jungles and oceans)
```suggestion
		addContinentalBiome(DUNES, OverworldClimate.DRY, 0.2, "dunes", config, enabledBiomes);
```